### PR TITLE
Displaying positive and negative offsets for edges in openlr assessment tool.

### PR DESCRIPTION
**Pull request by bykoianko**
Для корректной разметки необходимо видеть значение опциональных параметров olr:positiveOffset/olr:negativeOffset в утилите для разметки.

image

@vmihaylenko @gmoryes PTAL

### CODE REVIEW

File: openlr/openlr_match_quality/openlr_assessment_tool/traffic_mode.cpp [hunk 1]
```diff
@@ -230,6 +253,33 @@ QVariant TrafficMode::data(const QModelIndex & index, int role) const
...
+  if (index.column() == 3)
+    return m_segments[index.row()].GetNegativeOffset();
+
+  return QVariant();
+}
+
+QVariant TrafficMode::headerData(int section, Qt::Orientation orientation,
+                                 int role /* = Qt::DisplayRole */) const
+{
+  if (orientation == Qt::Horizontal)
```
Comment (vmihaylenko) [inline openlr/openlr_match_quality/openlr_assessment_tool/traffic_mode.cpp]: if (orientation != Qt::Horizonta || role != Qt::DisplayRole)
  return QVariant();

switch (section) 
{
case 0: return "Segment id";
case 1: return "Status code";
case 2: return "Positive offset (Meters)";
case 3: return "Negative offset (Meters)";
}

UNREACHABLE();

File: openlr/openlr_match_quality/openlr_assessment_tool/traffic_panel.cpp [hunk 2]
```diff
@@ -69,7 +69,7 @@ void TrafficPanel::CreateTable(QAbstractItemModel * trafficModel)
   m_table->setSelectionBehavior(QAbstractItemView::SelectionBehavior::SelectRows);
   m_table->setSelectionMode(QAbstractItemView::SelectionMode::SingleSelection);
   m_table->verticalHeader()->setVisible(false);
-  m_table->horizontalHeader()->setVisible(false);
+  m_table->horizontalHeader()->setVisible(true);
```
Comment (vmihaylenko) [inline openlr/openlr_match_quality/openlr_assessment_tool/traffic_panel.cpp]: inline comment please

File: openlr/openlr_match_quality/openlr_assessment_tool/traffic_mode.cpp (hunk 1 above)
Comment (bykoianko) [reply openlr/openlr_match_quality/openlr_assessment_tool/traffic_mode.cpp]: Done

File: openlr/openlr_match_quality/openlr_assessment_tool/traffic_panel.cpp (hunk 2 above)
Comment (bykoianko) [reply openlr/openlr_match_quality/openlr_assessment_tool/traffic_panel.cpp]: Are you sure we need it? The name of the method is like an inline comment, isn't it?
`m_table->verticalHeader()->setVisible(...);`, `m_table->horizontalHeader()->setVisible(...);`, `m_table->setShowGrid(...);`

**Comment by bykoianko** [comment]
@vmihaylenko PTAL

File: openlr/openlr_match_quality/openlr_assessment_tool/traffic_panel.cpp (hunk 2 above)
Comment (vmihaylenko) [reply openlr/openlr_match_quality/openlr_assessment_tool/traffic_panel.cpp]: indeed

**Pull request merged**